clients/privacy-center/__tests__/server-utils/recommendedSecurityHeaders.test.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  privacyCenterPagesCspHeader,
+  staticPageCspHeader,
+} from "~/app/server-utils/recommendedSecurityHeaders";
+
+describe("recommended security headers", () => {
+  it("templates and flattens the static csp header", () => {
+    expect(
+      staticPageCspHeader({
+        fidesApiHost: "fides.example.com",
+        geolocationApiHost: "geolocation.example.com",
+      }),
+    ).toMatchSnapshot();
+  });
+
+  it.each([true, false])(
+    "templates and flattens privacy center csp header",
```

Comment:
Consider outputting the current boolean value as the test name
```suggestion
    "templates and flattens privacy center csp header when isDev is %s,"
```